tensorflow/lite/mutable_op_resolver_test.cc

```diff
@@ -112,8 +122,34 @@ TEST(MutableOpResolverTest, FindCustomOp) {
   EXPECT_EQ(found_registration->builtin_code, BuiltinOperator_CUSTOM);
   EXPECT_TRUE(found_registration->invoke == DummyInvoke);
   EXPECT_EQ(found_registration->version, 1);
-  // TODO(ycling): The `custom_name` in TfLiteRegistration isn't properly
-  // filled yet. Fix this and add tests.
+}
+
+TEST(MutableOpResolverTest, FindCustomName) {
+  MutableOpResolver resolver;
+  resolver.AddCustom("AWESOME", GetDummyRegistration());
+
+  const TfLiteRegistration* found_registration = resolver.FindOp("AWESOME", 1);
+  ASSERT_NE(found_registration, nullptr);
+  EXPECT_EQ(found_registration->builtin_code, BuiltinOperator_CUSTOM);
+  EXPECT_TRUE(found_registration->invoke == DummyInvoke);
+  EXPECT_EQ(found_registration->version, 1);
+  EXPECT_EQ(found_registration->custom_name, "AWESOME");
+}
+
+TEST(MutableOpResolverTest, FindFunctionName) {
```

Comment:
This isn't really testing anything new.

Reply:
@jdduke thanks for the comments, i have removed this TC.

Regards
Amit